styles/styles.css

```
@@ -45,6 +54,7 @@ body {
 
 /*Шапка*/
 .page-header {
+  height: 66px;
```

```suggestion
  min-height: 66px;
```
старайся не принимать ничего фиксированного